timemachine/cpp/src/nonbonded_common.hpp

```diff
@@ -8,6 +8,9 @@
 
 namespace timemachine {
 
+// each atom parameterized by a 4-tuple: charge, lj sigma, lj epsilon, 4D coordinate w
+enum { PARAM_OFFSET_CHARGE = 0, PARAM_OFFSET_SIG, PARAM_OFFSET_EPS, PARAM_OFFSET_W, PARAMS_PER_ATOM };
```

Comment:
Moved this enum to a hpp file so we can access the values from the wrapped kernels. Seem like these values have utility beyond just the kernels.